htdocs/core/class/ldap.class.php

```diff
@@ -401,7 +396,7 @@ public function bindauth($bindDn, $pass)
 	public function unbind()
 	{
 		$this->result = true;
```

Comment:
this->connection can be a resource or (php8.1) a LDAP\Connection
Code must be compatible for both case. The new code seems ko with php 8.1. May be you can do 
`if (is_resource($this->connection) || is_object($this->connection)) {`